examples/clone/clone.bal

```diff
@@ -35,29 +35,29 @@ public function main() {
         address: address
     };
 
-    // Assign the `person` value to an `any` typed variable called `anyValue`.
-    // The type of the value held by `anyValue` is determined at runtime.
+    // Assigns the `person` value to an `any`-typed variable called `anyValue`.
+    // The type of the value held by `anyValue` is determined during runtime.
     any anyValue = person;
 
-    // Invoke the `.clone()` built-in method. The return type is either `Person` or `error`. Note that `error` could be
-    // returned if the variable on which `.clone()` is called is not of type `anydata`.
-    // If the type of the value that is cloned can be determined at compile time and is `anydata`, the type of the
+    // Invokes the `.clone()` built-in method. The return type is either `Person` or `error`. `error` could be
+    // returned if the variable on which `.clone()` is called is not of the type `anydata`.
+    // If the type of the value that is cloned can be determined during compile time and is `anydata`, the type of the
     // return value is exactly the type of the value being cloned.
-    // e.g. `Person result = person.clone();`
-    // If `person` is cloned instead of `anyValue`, at compile time the compiler will determine that the return type is
+    // E.g., `Person result = person.clone();`
+    // If `person` is cloned instead of `anyValue`, during compile time the compiler will determine that the return type is
     // `Person`.
     var result = anyValue.clone();
 
-    // The type of the value held by an `any` typed variable is only determined at runtime.
-    // Therefore, the type of the returned value on a clone attempt could be `error`, in case a non-`anydata` value is
+    // The type of the value held by an `any`-typed variable is only determined during runtime.
+    // Therefore, the type of the returned value on a clone attempt could be `error`, in case if a non-`anydata` value is
```

Comment:
Do we the "if" here? Doesn't read right IMO.

We can remove "in case" and just say "if", if just "in case" is not OK.

Reply:
It is fine to say "in case if...".

Reply:
Isn't it redundant? Can't we just say
```suggestion
    // Therefore, the type of the returned value on a clone attempt could be `error`, if a non-`anydata` value is
```